AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Commands/AWSSDKSwiftCLI/Subcommands/GenerateDocIndex.swift

```diff
@@ -7,6 +7,7 @@
 
 import ArgumentParser
 import Foundation
+import AWSCLIUtils
```

Comment:
Many utility types & functions were split into a new `AWSCLIUtils` module so they could be shared between the old `AWSSDKSwiftCLI` tool and the new registry commands.

Hence, a lot of imports to use that module.